api/proto/config/appconfig.proto

```diff
@@ -26,6 +26,47 @@ enum MetaDataType {
   MetaDataDriveMultipart = 3; // Process multipart MIME for application
 }
 
+// The information about a snapshot
+message AvailableSnapshot {
+  string id = 1;
+}
+
+// The snapshot information for an Application Instance
+message SnapshotConfig {
+  // activeSnapshot is the id of the snapshot which is currently used.
```

Comment:
"is currently used" sounds like status and not config. 
Is this a declarative statement saying that the application instance should be using this snapshot?

Reply:
Together with the rollback_counter field - it triggers the rollback process.
I can reword it to "is expected to be used for the given app"